src/EditorFeatures/VisualBasic/LineCommit/ICommitFormatter.vb

```diff
@@ -12,13 +11,13 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
         ''' by this function. Further, if the operation is cancelled, the buffer may be left in a
         ''' partially committed state that must be rolled back by the transaction.
         ''' </summary>
-        Function CommitRegionAsync(
+        Sub CommitRegion(
```

Comment:
I've searched their repository and confirmed that they do not implement this.  This interface is only used by Visual Basic